tools/witx/src/lib.rs

```diff
@@ -33,6 +38,14 @@ pub fn load<P: AsRef<Path>>(path: P) -> Result<Document, WitxError> {
     validate_document(&parsed_decls).map_err(WitxError::Validation)
 }
 
+pub fn parse(source: &str) -> Result<Document, WitxError> {
```

Comment:
I assume this only works on strings that don't contain `(use ...)` directives? If so, that'd be good to mention in a comment.